### ISSUE TITLE
Darktable takes about a minute to start up and every parameter change takes about a second or two to actually alter the image

### ISSUE THREAD

**Issue by Robotomo1** [darktable-org/darktable#17554]
### Describe the bug

Darktable 4.8.1 and Nightly 4.9.0 take about a minute to start up from first clicking on it. After such a long wait time to start the program, everything that I do inside version 4.8.1 specifically is slow. Going from the lighttable to the darkroom and changing various parameters all take about a second or two to actually do something. For example, I raise the exposure and have to wait a second or two for this raised exposure to actually be applied to the image. I'm only loading about a handful of image too, so it's not like Darktable is trying to load thousands of images.
Nightly 4.9.0 however, despite it taking a minute to start, the parameter changes are almost instant. 

In the log file, "'G_IS_OBJECT (object)' failed" seems to be causing the slow start. 

Trying Darktable 4.8.1 on two different Windows 10 and Windows 11 machines does not yield the same results as on my machine.

### Steps to reproduce

1 - Launch Darktable 4.8.1 or Nightly 4.9.0
2 - Edit a photo as you normally would. 4.8.1 takes a second or two to actually register every edit I make. Moving the exposure up for example takes about a second or two to actually show on the photo I'm editing

### Expected behavior

Darktable should launch in less than ten seconds and each parameter change should be almost instant.

### Logfile | Screenshot | Screencast

Here's the pastebin link to the logs: https://pastebin.com/MbehpPYp 

In the log file, "'G_IS_OBJECT (object)' failed" seems to be causing the slow start. 

### Where did you obtain darktable from?

4.8.1 downloaded from www.darktable.org
nightly 4.9.0 downloaded from github

### darktable version

4.8.1

### What OS are you using?

Windows

### What is the version of your OS?

Windows 10 Home

### Describe your system?

- Ryzen 5 3600
- 16gb ram
- GTX 1050ti 4gb

### Are you using OpenCL GPU in darktable?

Yes

### If yes, what is the GPU card and driver?

GTX 1050ti 4gb. Driver version 561.09

### Please provide additional context if applicable. You can attach files too, but might need to rename to .txt or .zip

Darktable 4.2.1 and 4.0.0 fail to launch.
Darktable nightly 4.9.0 launches slowly, but paramater changes are instant.

**Comment by ralfbrown** [comment]
>  "'G_IS_OBJECT (object)' failed" seems to be causing the slow start.

No, that error message takes essentially no time and is just a symptom of some other problem with the GUI.  Opening the two database files is taking over 40 seconds, which shouldn't be the case unless you have hundreds of thousands of images in your libraries *and* are storing the databases on a slow/network drive (I just checked - my library of 370k images takes 12 seconds to open with the database stored on SSD).

darktable relies on sqlite3 for database access.  Perhaps you have a misconfigured sqlite DLL?

**Comment by Robotomo1** [comment]
I see now. Thank you for the quick and easy to understand reply!

Sorry, but I have no knowledge about sqlite3. What do you recommend I look for to see if I have a misconfigured sqlite DLL?

**Comment by ralfbrown** [comment]
Not a Windows user and not a sqlite expert, so I have no idea....
